File: exercises/JavaScript/chapter_1/6_sales_tax.js
// 6. Sales Tax
// Design a Python program that will ask the user to enter the amount of a purchase. The program should then compute
// the VAT tax. The program should display the amount of the purchase, the VAT tax, and the total of the sale (which is the
// sum of the amount of purchase plus the VAT tax).
// Hint: Use the value 0.15 to represent 15%.

const amount_of_a_purchase = 18450
const vat_tax = 0.15 * amount_of_a_purchase
const total_sale = amount_of_a_purchase + vat_tax

console.log("Amount of a purchase:", amount_of_a_purchase)
console.log("VAT Tax:", vat_tax)
console.log("Total Sale:", total_sale)

File: exercises/C/chaper_1 - Basics/6_sales_tax.c
// 6. Sales Tax
// Design a Python program that will ask the user to enter the amount of a purchase. The program should then compute
// the VAT tax. The program should display the amount of the purchase, the VAT tax, and the total of the sale (which is the
// sum of the amount of purchase plus the VAT tax).
// Hint: Use the value 0.15 to represent 15%.
#include <stdio.h>


int main(void)
{
    double amount_of_a_purchase;
    double vat_tax;
    double total_sale;

    printf("Enter total of a purchase: ");
    scanf("%lf", &amount_of_a_purchase);

    vat_tax = 0.15 * amount_of_a_purchase;
    total_sale = vat_tax + amount_of_a_purchase;

    printf("Total Purchase: %.2lf\n", amount_of_a_purchase);
    printf("Vat Tax: %.2lf\n", vat_tax);
    printf("Total Sale: %.2lf\n", total_sale);
}